### ISSUE TITLE
Wrong text in update profile privacy

### ISSUE THREAD

**Issue by B3nito** [joomla/joomla-cms#22174]
### What needs to be fixed
You have registration form and you have update profile form.
Both form show text: 

PLG_SYSTEM_PRIVACYCONSENT_NOTE_FIELD_DEFAULT="By signing up to this web site and agreeing to the Privacy Policy you agree to this web site storing your information."

But in update profile I not sign up.

There is string:
PLG_USER_TERMS_NOTE_FIELD_DEFAULT="By signing up to this web site you accept the Terms & Conditions."

is it using in term field. but where is term field and text setup?

### Why this should be fixed
In update user not sign up, can be misunderstanding, must be different string.

**Comment by ghost** [comment]
can a experienced User please answer as the Issue is open since one Week having Status "New"?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22174">issues.joomla.org/tracker/joomla-cms/22174</a>.</sub>

**Comment by ghost** [comment]
i'll try calling @infograf768

**Comment by infograf768** [comment]
The confusion is rather for Terms of Service and Terms & Conditions.
My advise is to choose one or the other but not both.

![screen shot 2019-03-05 at 12 37 03](https://user-images.githubusercontent.com/869724/53802923-6ddbf080-3f43-11e9-8bc0-b4b35d3b7064.png)

The User Profile plugin, when enabled and if the param is required and no article is set is, blatantly displaying Agree or No without any information
![screen shot 2019-03-05 at 12 37 57](https://user-images.githubusercontent.com/869724/53803067-d3c87800-3f43-11e9-8020-789654837514.png)

The User Terms and Conditions plugin, if enabled and no article is set, is at least displaying the default Short Term and Conditions.

This PLG_USER_TERMS_NOTE_FIELD_DEFAULT string  may be modified by a language override.
![screen shot 2019-03-05 at 12 51 09](https://user-images.githubusercontent.com/869724/53803562-61f12e00-3f45-11e9-8db3-913df1e945ab.png)

to obtain:
![screen shot 2019-03-05 at 12 51 59](https://user-images.githubusercontent.com/869724/53803611-81885680-3f45-11e9-94f3-e98bbfdf2cb9.png)

In a similar way one can change the default Short Privacy Policy by OR entering a new text in the System Privacy plugin parameter OR an override of the string concerned (PLG_SYSTEM_PRIVACYCONSENT_NOTE_FIELD_DEFAULT).

Basically, the existing strings are quite neutral and can easily be overriden or, in a single language site, modified directly in the plugin params.

**Comment by ghost** [comment]
thanks for Help @infograf768, so closing the Issue.
